get_involved.md

```diff
@@ -6,30 +6,48 @@ layout: default
+hep-sf-forum+subscribe@googlegroups.com
 
-In addition, there is a HEP S&C community mailing list for occasional HEP software and computing community mailings. Everyone involved or interested in HEP S&C is encouraged to sign up to this list. It will be used for occasional mailings of community wide interest, such as announcements of HEP S&C conferences, workshops and schools.
+In addition, there is a HEP S&C community mailing list for occasional HEP
+software and computing community mailings. Everyone involved or interested in
+HEP S&C is encouraged to sign up to this list. It will be used for occasional
+mailings of community wide interest, such as announcements of HEP S&C
+conferences, workshops and schools.
 
-# Areas for contribution 
+## Areas for contribution
 
-There are plenty of opportunities for you to contribute to the HSF, be it by participation in technical discussions, by joining one of our working groups, or by joining the HSF with your software project:
+There are plenty of opportunities for you to contribute to the HSF, be it by
+participation in technical discussions, by joining one of our working groups, or
+by joining the HSF with your software project:
 
   *  Particpation in HSF activities and fora. See the Activities page.
-  *  Participation in HSF events and meetings. 
+  *  Participation in HSF events and meetings.
   *  Taking advantage for your project from the HSF (to be created)
   *  Giving input to the HSF startup team
   *  Contributing to this website
 
-# Meetings
+## Meetings
 
-See the Events listing on this site for information on HSF meetings, workshops, etc. Meetings use CERN indico for agenda pages. The pages are open to the world, no CERN account needed. So is Vidyo, the conference system we use. Connection info is on the indico pages. However should you need to upload material to the agenda, a CERN account is needed. CERN provides lightweight accounts to non CERN affiliated people when needed. See https://account.cern.ch/account/Externals/
+See the Events listing on this site for information on HSF meetings, workshops,
+etc. Meetings use CERN indico for agenda pages. The pages are open to the world,
+no CERN account needed. So is Vidyo, the conference system we use. Connection
+info is on the indico pages. However should you need to upload material to the
+agenda, a CERN account is needed. CERN provides lightweight accounts to non CERN
+affiliated people when needed. See https://account.cern.ch/account/Externals/
 
-# Giving input to the HSF
+## Giving input to the HSF Startup Team
 
-In this document we provide background information, guidelines and a questionnaire for giving input to the HSF, and as background to discussions with the HSF startup team. We'd love to hear from you. If you'd like to arrange a discussion, contact any member of the startup team or email the startup team at hep-sf-startup-team@googlegroups.com.
+If the existing [fora](/forums.html) do not seem to be the right place to ask
```

Comment:
I'd say: `If you are not confortable with raising an issue in the [forums](/forums.html] or if you are afraid that this is not the right place for it, feel free...`